package.json

```diff
@@ -14,7 +14,7 @@
         "lint": "eslint src tests --fix",
         "integration-test": "echo 'Browse to localhost:3000/tests' && node testServer.js",
         "prepublish": "yarn test && yarn build && yarn build-module && yarn process-types",
-        "test": "jest",
+        "test": "nodemon --ext js --exec 'yarn jest'",
```

Comment:
Hm, wouldn't `jest` with the `--watch` options do the job here?

Reply:
Ah, I didn't know that was an option! Updated